topostats/default_config.yaml

```
@@ -48,5 +48,6 @@ plotting:
   zrange: [null, null]  # low and high height range for core images (can take [null, null])
   colorbar: true  # Options : true, false
   axes: true # Options : true, false (due to off being a bool when parsed)
-  cmap: nanoscope # Options : nanoscope, afmhot
+  cmap: nanoscope # Options : nanoscope, afmhot, gwyddion
+  mask_cmap: blu # Options : blu, jet_r and any in matplotlib
```

I might have missed it but would it be worth adding a `dpi` option to the `default_config.yaml`?

Otherwise users would have to load all the classes and modules into an interactive Python session, process their images and then instantiate the `Images()` class with the desired `dpi` option.

```suggestion
  mask_cmap: blu # Options : blu, jet_r and any in matplotlib
  dpi: figure # Options figure or int (integer) representing the dots per inch
```

---

So I thought it be best not to, in order to keep the config less complex, and instead add it into the plotting dictionary as this really only applies to the skeletons. what are your thoughts?